roles/configure_ztp_gitops_apps/tasks/main.yaml

```diff
@@ -24,68 +24,89 @@
         mode: '0775'
 
     - name: "Create a pull-secret file for podman"
-      copy:
+      ansible.builtin.copy:
```

Comment:
The `ocp_pull_secret` is a variable that is required for this role, this should be placed in the README + prefix it with `ztp_`

Reply:
Done.